packages/web3torrent/src/clients/web3torrent-client.ts

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import WebTorrentPaidStreamingClient from '../library/web3torrent-lib';
+import {Torrent} from '../types';
+
+export const web3torrent = new WebTorrentPaidStreamingClient();
+export const WebTorrentContext = React.createContext(web3torrent);
+
+export const download: (torrent: any) => Promise<Torrent> = torrentData =>
+  new Promise(resolve =>
+    web3torrent.add(torrentData, torrent => resolve({...torrent, status: 'Connecting'}))
+  );
+
+export const upload = files =>
+  new Promise(resolve => web3torrent.seed(files as FileList, torrent => resolve(torrent)));
+
+export const remove = (id: string) => {
+  return new Promise((resolve, reject) =>
+    web3torrent.remove(id, err => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(id);
+      }
+    })
+  );
+};
+
+const getStatus = (torrent: Torrent) => {
+  const {uploadSpeed, downloadSpeed} = torrent;
+  if (torrent.downloaded && torrent.downloaded === torrent.length && torrent.done) {
```

Comment:
No need to use `else if` in any of these. Just a sequence of `ifs`, the first one that resolves the condition will make the function exit.